intermediate_source/named_tensor_tutorial.py

```diff
@@ -0,0 +1,518 @@
+# -*- coding: utf-8 -*-
+"""
+Introduction to Named Tensors in PyTorch

```

Comment:
We should note that it is experimental somewhere. I'm not sure where the best place for that is though.

Reply:
Hmm, I'll ping Joe Spisak & co. about whether to include the word experimental in the title here.

Reply:
Update: Soumith feels we should have "experimental" in the title - good call.